internal/global/trace_test.go

```diff
@@ -91,3 +101,101 @@ func TestTraceProviderDelegates(t *testing.T) {
 	gtp.Tracer("abc", trace.WithInstrumentationVersion("xyz"))
 	assert.True(t, called, "expected configured TraceProvider to be called")
 }
+
+func TestTraceProviderDelegates_MultiGoRoutines(t *testing.T) {
+	global.ResetForTest()
+
+	// Retrieve the placeholder TracerProvider.
+	gtp := otel.GetTracerProvider()
+
+	done := make(chan int)
+	quit := make(chan int)
```

Comment:
Any reason to not make these `chan struct{}`?

Reply:
Done.